src/services/state-persistor/state-persistor.model.ts

```diff
@@ -20,14 +21,26 @@ export const SP_KEY_Y = 'Y'
 export enum SP_AVAILABLE_STORAGES {
   localStorage = 0,
   permalink = 1,
-  both = 2,
+  permalinkAsPath = 2,
+  both = 3,
```

Comment:
Maybe rename `both` as it has become less clear?

Reply:
Done.